yarn-project.nix

```diff
@@ -1246,6 +1246,7 @@ cacheEntries = {
 "@lerna/symlink-dependencies@npm:5.2.0" = { filename = "@lerna-symlink-dependencies-npm-5.2.0-a6aa998578-4a00e85ac8.zip"; sha512 = "4a00e85ac8bd63896834978cd6d380fecb4093098f09e7d53112b4543ea3ab70b563c24467748c6a4d1e77a25cb0a463f2a3284339bf7366eacff9f1d2ca6593"; };
 "slice-ansi@npm:3.0.0" = { filename = "slice-ansi-npm-3.0.0-d9999864af-5ec6d022d1.zip"; sha512 = "5ec6d022d12e016347e9e3e98a7eb2a592213a43a65f1b61b74d2c78288da0aded781f665807a9f3876b9daa9ad94f64f77d7633a0458876c3a4fdc4eb223f24"; };
 "minipass-fetch@npm:2.1.0" = { filename = "minipass-fetch-npm-2.1.0-300ce55188-1334732859.zip"; sha512 = "1334732859a3f7959ed22589bafd9c40384b885aebb5932328071c33f86b3eb181d54c86919675d1825ab5f1c8e4f328878c863873258d113c29d79a4b0c9c9f"; };
+"fsevents@patch:fsevents@npm%3A2.3.2#~builtin<compat/fsevents>::version=2.3.2&hash=18f3a7" = { filename = "fsevents-patch-3340e2eb10-8.zip"; sha512 = "edbd0fd80be379c14409605f77e52fdc78a119e17f875e8b90a220c3e5b29e54a1477c21d91fd30b957ea4866406dc3ff87b61432d2840ff8866b309e5866140"; };
```

Comment:
I wonder why this change is here? 🤔

Reply:
thanks @mirceahasegan , a refuse... 0c1b5d5bc should resolve this.